int64 id
geometry_msgs/Point position
float32 bid
VoteList[] vote_list
float32 score
int16 role